wallet-new/src/Cardano/Wallet/Kernel/DB/TxMeta/Types.hs

```diff
@@ -79,7 +79,7 @@ data TxMeta = TxMeta {
     , _txMetaAmount     :: Core.Coin
 
       -- | Transaction inputs
-    , _txMetaInputs     :: NonEmpty (Core.Address, Core.Coin, Txp.TxId, Word32)
+    , _txMetaInputs     :: NonEmpty (Txp.TxId, Word32, Core.Address, Core.Coin)
```

Comment:
@edsko I thought a bit about this. TxMeta, should not be uniquely identified by TxId, but from the pair (TxId, Account). The Meta may be different for two accounts and we should keep both Meta. I think this should not change, what do you think?

Reply:
Assume an Account sends coins to another Account on the same wallet instance. The Tx will have different TxMeta for the two Accounts, yet it will be the same txId. We need to save both TxMeta as a list [TxMeta] and not as a Map TxMeta -> [Account].

Reply:
Yes, point taken. I agree.